### PR TITLE
feat: add getPreviousMessagesV2 and getRecentMessagesV2 

### PR DESCRIPTION
**Pull request by MocA-Love**
love

### CODE REVIEW

File: packages/client/clients/internal/talk-client.ts [hunk 1]
```diff
@@ -279,11 +279,38 @@ export class TalkClient extends ChannelClient {
...
+			this.TalkService_API_PATH,
+		)).map((e: LooseType) => this.parser.rename_thrift("Message", e));
+	}
+
 	/**
 	 * @description Get user information from mid.
 	 */
 	public async getContact(options: {
 		mid: string;
+		time
```
Comment (EdamAme-x) [inline packages/client/clients/internal/talk-client.ts:313]:
```suggestion

```
Don't need this line

**Review by EdamAme-x — CHANGES_REQUESTED** [review]

**Comment by MocA-Love** [comment]
gogo

**Comment by MocA-Love** [comment]
add getRecentMessagesV2

**Review by EdamAme-x — APPROVED** [review]

**Comment by EdamAme-x** [comment]
LGTM!

**Pull request merged**